### ISSUE TITLE
CustomStat de l'index

### ISSUE THREAD

**Issue by lpofredc** [PnEcrins/GeoNature-atlas#144]
Sur d'importants jeux de données, plusieurs millions, customStat de la fonction index ralentit énormément le chargement de la page alors que l'option AFFICHAGE_RANG_STAT n'est pas activée. la donnée est cachée dans le template mais éxécutée tout de même éxécutée côté serveur.

Proposition d'éxécution de la vue sous condition semblable au template:

```python
def index():
    ...
    customStatMedias = vmObservationsRepository.genericStatMedias(
        connection, config.RANG_STAT
    )
    if customStatMedias != None and config.AFFICHAGE_RANG_STAT:
        customStat = vmObservationsRepository.genericStat(connection, config.RANG_STAT)
    else:
        customStat = None
```

**Comment by camillemonchicourt** [comment]
Bien vu.

**Comment by Adrien-Pajot** [comment]
@corentinlange

@jpm-cbna , est-ce que tu as traité ce problème de perf dans tes modifs ?

**Comment by camillemonchicourt** [comment]
Pour moi c'est dans cette PR : https://github.com/PnX-SI/GeoNature-atlas/pull/276

**Comment by jpm-cbna** [comment]
@Adrien-Pajot Oui, je pense que c'est réglé [l94-97](https://github.com/PnX-SI/GeoNature-atlas/pull/276/files#diff-5ab357ab30a3a6fa44a94aeafbd6871ae89d65e098f76d8e4cd9736116685e9bR94-L93).
